code/game/objects/items/weapons/defib.dm

```diff
@@ -162,7 +171,7 @@
 					to_chat(user, "<span class='warning'>[src] buzzes: Defibrillation failed. Vital signs are too weak, please try again in five seconds.</span>")
 					return
 			//we couldn't find a suitable ghost.
-			target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. Patient's condition does not allow reviving.</span>")
+			target.visible_message("<span class='warning'>[src] buzzes: Defibrillation failed. Unable to diagnose reason - body suitable for operation.</span>")
```

Comment:
I mean we have literal soul detection trays, is it really too out of character to just flat out say "No soul detected?"

![image](https://user-images.githubusercontent.com/9782036/35782074-45de130e-09b8-11e8-934d-641775740df1.png)

Reply:
That's a good point, a few months back, you could examine corpses and it would tell you that THEIR SOUL HAS DEPARTED (IE: braindead). Dunno why that went away.

Reply:
I'm not convinced this is a good idea.
The "soul has departed" message is adressed to humans ; the defib is a machine.
The morgue green lights don't have such a blunt way to present it.

Reply:
Give "no brain waves detected" for that and the other thing should be "central nervous system not found" or something.